frontend/src/components/app/app.tsx

```diff
@@ -1,31 +1,13 @@
 import { FC } from 'react';
-import logo from 'assets/img/logo.svg';
-import { Link, Routes, Route } from 'components/common/common';
-import { useLocation } from 'hooks/hooks';
+import { Routes, Route } from 'components/common/common';
 import { AppRoute } from 'common/enums/enums';
 import { Auth } from 'components/auth/auth';
+import styles from './app.module.scss';
 
 const App: FC = () => {
-  const { pathname } = useLocation();
-
   return (
     <>
-      <img src={logo} className="App-logo" width="30" alt="logo" />
-
-      <ul className="App-navigation-list">
-        <li>
-          <Link to={AppRoute.ROOT}>Root</Link>
-        </li>
-        <li>
-          <Link to={AppRoute.SIGN_IN}>Sign in</Link>
-        </li>
-        <li>
-          <Link to={AppRoute.SIGN_UP}>Sign up</Link>
-        </li>
-      </ul>
-      <p>Current path: {pathname}</p>
-
-      <div>
+      <div className={styles.authConteiner}>
```

Comment:
I think its better not to add styling to `App` component itself. If you want to apply styles to all pages please create separate `<Page/>` component with needed styles